apps/io-wallet-user-func/src/app/config.ts

```diff
@@ -99,13 +99,26 @@ export type TrialSystemApiClientConfig = t.TypeOf<
   typeof TrialSystemApiClientConfig
 >;
 
+const PidIssuerApiClientConfig = t.type({
+  baseURL: t.string,
+  clientCertificate: t.string,
+  clientPrivateKey: t.string,
+  rootCACertificate: t.string,
+  walletProviderEntity: t.string,
```

Comment:
This information (`walletProviderEntity`) is generic for the entire backend. You can find it already valued in: `FederationEntityBasePath` env var.